pom.xml

```diff
@@ -184,10 +185,25 @@
         <optional>true</optional>
       </dependency>
       <dependency>
-        <groupId>io.reactivex</groupId>
+        <groupId>io.reactivex.rxjava2</groupId>
         <artifactId>rxjava</artifactId>
         <version>${rxjava.version}</version>
       </dependency>
+      <dependency>
```

Comment:
This is actually a temporary hack, for https://github.com/vert-x3/vertx-codegen/issues/143.
Maybe we should raise a ticket in knot.x to remember to fix that?